react/features/toolbox/components/native/Toolbox.js

```diff
@@ -87,30 +92,33 @@ function Toolbox(props: Props) {
                 edges = { [ bottomEdge && 'bottom' ].filter(Boolean) }
                 pointerEvents = 'box-none'
                 style = { styles.toolbox }>
-                <AudioMuteButton
+                {!_visitorsModeEnabled && <AudioMuteButton
                     styles = { buttonStylesBorderless }
                     toggledStyles = { toggledButtonStyles } />
-                <VideoMuteButton
+                }
+                {!_visitorsModeEnabled && <VideoMuteButton
                     styles = { buttonStylesBorderless }
                     toggledStyles = { toggledButtonStyles } />
-                {
-                    additionalButtons.has('chat')
+                }
+                {!_visitorsModeEnabled && additionalButtons.has('chat')
                       && <ChatButton
                           styles = { buttonStylesBorderless }
                           toggledStyles = { backgroundToggledStyle } />
                 }
-                {additionalButtons.has('screensharing') && <ScreenSharingButton styles = { buttonStylesBorderless } />}
-                { additionalButtons.has('raisehand') && (_reactionsEnabled
+                {!_visitorsModeEnabled && additionalButtons.has('screensharing')
+                    && <ScreenSharingButton styles = { buttonStylesBorderless } />}
+                { !_visitorsModeEnabled && additionalButtons.has('raisehand') && (_reactionsEnabled
                     ? <ReactionsMenuButton
                         styles = { buttonStylesBorderless }
                         toggledStyles = { backgroundToggledStyle } />
                     : <RaiseHandButton
                         styles = { buttonStylesBorderless }
                         toggledStyles = { backgroundToggledStyle } />)}
                 {additionalButtons.has('tileview') && <TileViewButton styles = { buttonStylesBorderless } />}
-                <OverflowMenuButton
+                {!_visitorsModeEnabled && <OverflowMenuButton
```

Comment:
Shoudln't they be able to use the overflow button to get to settings to change their name for example?

Reply:
Well ... there is no point as nobody will be seeing it.

Reply:
There are many other options like enabling close captions for example. What about those?

"We can look into that later" is also a valid answer ;-)

Reply:
Yep. For sure there will be no cc for visitors for now ...